tests/stdc/enc/enc.cpp

```diff
@@ -224,6 +223,33 @@ static void _test_time_functions(void)
 
         OE_TEST(after > before);
     }
+
+    /* Test nanosleep() with req==rem */
+    {
+        uint64_t before = oe_get_time();
+
+        /* Sleep for SLEEP_SECS seconds */
+        {
+            timespec ts = {SLEEP_SECS, 0};
+            OE_TEST(nanosleep(&ts, &ts) == 0);
```

Comment:
Should this also check if ts is zero'd out?

Reply:
I'm not sure. The real nanosleep does not change rem if it has not been interrupted.